WeatherApp/Globals/UIExtensions.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  UISetup.swift
+//  WeatherApp
+//
+//  Created by Lucija Balja on 06/08/2020.
+//  Copyright © 2020 Lucija Balja. All rights reserved.
+//
+
+import UIKit
+
+extension UILabel {
+    
+    func style(size: CGFloat = 20.0, textAlignment: NSTextAlignment = .center) {
+        self.font = .boldSystemFont(ofSize: size)
+        self.textColor = .white
+        self.textAlignment = textAlignment
+        self.clipsToBounds = true
+        self.adjustsFontSizeToFitWidth = true
+        self.translatesAutoresizingMaskIntoConstraints = false
+    }
+}
```

Comment:
I would prefer to have extensions of different classes in separate files. 
You can call the file with UILabel extensions **UILabel+Extensions.swift** 
@istaudacher Correct me if I'm wrong on the naming. I'm not sure if we have a different guideline for that.

Reply:
@dorastipkovic @termodinamika Correct, extension file for class XYZ should be called XYZ+Extensions.swift.